tests/test_ax_integration.py

```diff
@@ -113,6 +118,127 @@ def test_generation_method(self):
         self.assertEqual(n_sobol, correct_n_sobol)
         self.assertEqual(n_opt, correct_n_opt)
 
+class AxOrdinalGPTestCase(unittest.TestCase):
+    """
+    Tests that the Ax integration works with an ordinal GP model.
+    """
+    @classmethod
+    def setUpClass(cls):
+
+        # Fix random seeds
+        np.random.seed(int(time.time()))
```

Comment:
The random seed should be fixed to some fixed value like 0 or 1. The purpose of this is to be able to reproduce the same output every time you run the code.